source/access/mfa_login.rst

```diff
@@ -3,23 +3,221 @@
 Multi Factor Authentication (MFA)
 =================================
 
-|KUL| Multi Factor Authentication (MFA) is an augmented level of security which, as
-the name suggests, requires multiple steps when authenticating. The following
-two factors are necessary to connect to the KU Leuven clusters:
+|KUL| Multi Factor Authentication (MFA) is an augmented level of security.
+As the name suggests, MFA requires additional steps with human intervention
+when authenticating.
+MFA is mandatory for accessing KU Leuven infrastructures.
+This document provides two methods to connect to the KU Leuven clusters with MFA.
 
-- A valid private key
-- Access to a VSC-associated university/institution account
+The first method, *without* an :ref:`SSH agent <SSH agent>`, is easier to get started with,
+but requires some repetitive steps with each connection, hence inconvenient on a long run.
 
-This approach ensures that if your private key is compromised, the person who
-has unauthorized access to it will still not be able to abuse your VSC account.
-This document provides two methods to connect to the KU Leuven clusters with
-MFA. The first method, *without* an agent, is easier to get started with, but requires
-some repetitive steps with each connection. The second method, *with* an agent,
-requires some more effort to set up initially but will be easier when you need
-multiple connections or you are using some specific GUI applications to connect. 
+The second method involves an :ref:`SSH agent <SSH agent>` and requires a few more
+steps to set up. This approach is however more convenient in the long run because
+a single authentication will suffice for multiple connections throughout the day,
+without further intervention.
 
-.. toctree::
+Below, we elaborate on these two approaches.
 
-  mfa_quickstart
-  mfa_guide
+.. note::
+
+   When connecting from abroad, you first need to login via the `VSC firewall page`_.
+
+.. _mfa quick start:
+
+Connecting without an SSH agent
+-------------------------------
+
+First of all, choose whether you want to login to the `VSC firewall page`_,
+:ref:`Open OnDemand portal <ood_t2_leuven>`, or to the :ref:`Tier-2 login <tier2_login_nodes>`
+page using a text-based terminal (such as PuTTY or MobaXterm).
+
+.. tab-set::
+
+   .. tab-item:: VSC firewall page
+
+      Proceed to the `VSC firewall page`_ on your browser.
+
+   .. tab-item:: Open OnDemand
+
+      Proceed to the :ref:`Open OnDemand portal <ood_t2_leuven>`.
+      If you are affiliated with KU Leuven, click on the KU Leuven logo.
+      Otherwise, click on the VSC logo to proceed to the authentication page.
+
```

Comment:
Also KU Leuven users can login through the VSC login. So I would suggest to let all users click on this logo.

Reply:
I agree with you, but there is yet one more benefit in using the KUL logo for the KUL users. If in their browsers they already had logged in to internal KUL services and have already gone through MFA, then by clicking on the KUL logo, they immediately land on the ondemand page, and it saves them some extra/unnecessary clicks needed with the VSC logo.
So, I rather keep it as it is.